codice-itest-impl/src/main/java/org/codice/itest/reporter/LoggingDiagnosticTestReporter.java

```diff
@@ -36,6 +36,7 @@ public void accept(TestResult testResult) {
             case FAIL: logger.warn(formattedResult);
                 break;
             case ERROR: logger.error(formattedResult);
+            case NOT_EXECUTED: logger.error(formattedResult);
```

Comment:
✏️ A nit but you can just add this to the error case.
```suggestion
            case ERROR, NOT_EXECUTED: logger.error(formattedResult);
```